lib/data/2024_hud_lists.json

```diff
@@ -0,0 +1,4479 @@
+        "description": "Tenant-based - scattered site"
+      }
+    ]
+  },
+  {
+    "name": "ProjectType",
+    "code": "2.02.6",
+    "values": [
+      {
+        "key": 0,
+        "description": "Emergency Shelter - Entry Exit"
+      },
+      {
+        "key": 1,
+        "description": "Emergency Shelter - Night-by-Night"
+      },
+      {
+        "key": 2,
+        "description": "Transitional Housing"
+      },
+      {
+        "key": 3,
+        "description": "PH - Permanent Supportive Housing"
+      },
+      {
+        "key": 4,
+        "description": "Street Outreach"
+      },
+      {
+        "key": 6,
+        "description": "Services Only"
+      },
+      {
+        "key": 8,
+        "description": "Safe Haven"
+      },
+      {
+        "key": 9,
+        "description": "PH - Housing Only"
+      },
+      {
+        "key": 7,
+        "description": "Other"
+      },
+      {
+        "key": 10,
+        "description": "PH - Housing with Services (no disability required for entry)"
+      },
+      {
+        "key": 11,
+        "description": "Day Shelter"
+      },
+      {
+        "key": 12,
+        "description": "Homelessness Prevention"
+      },
+      {
+        "key": 13,
+        "description": "PH - Rapid Re-Housing"
+      },
+      {
+        "key": 14,
+        "description": "Coordinated Entry"
+      }
+    ]
+  },
+  {
+    "name": "RRHSubType",
+    "code": "2.02.A",
+    "values": [
+      {
+        "key": 1,
+        "description": "RRH: Services Only"
+      },
+      {
+        "key": 2,
+        "description": "RRH: Housing with or without services"
+      }
+    ]
+  },
+  {
+    "name": "FundingSource",
+    "code": "2.06.1",
+    "values": [
+      {
+        "key": 1,
+        "description": "HUD: CoC - Homelessness Prevention (High Performing Communities Only)"
+      },
+      {
+        "key": 2,
+        "description": "HUD: CoC - Permanent Supportive Housing"
+      },
+      {
+        "key": 3,
+        "description": "HUD: CoC - Rapid Re-Housing"
+      },
+      {
+        "key": 4,
+        "description": "HUD: CoC - Supportive Services Only"
+      },
+      {
+        "key": 5,
+        "description": "HUD: CoC - Transitional Housing"
+      },
+      {
+        "key": 6,
+        "description": "HUD: CoC - Safe Haven"
+      },
+      {
+        "key": 7,
+        "description": "HUD: CoC - Single Room Occupancy (SRO)"
+      },
+      {
+        "key": 8,
+        "description": "HUD: ESG - Emergency Shelter (operating and/or essential services)"
+      },
+      {
+        "key": 9,
+        "description": "HUD: ESG - Homelessness Prevention "
+      },
+      {
+        "key": 10,
+        "description": "HUD: ESG - Rapid Rehousing"
+      },
+      {
+        "key": 11,
+        "description": "HUD: ESG - Street Outreach"
+      },
+      {
+        "key": 12,
+        "description": "HUD: Rural Housing Stability Assistance Program",
+        "deprecated": true
```

Comment:
I'm hesitating to remove some of these old ones.  The mapping file says "no mapping required" so what happens to the funders we have that have source 12? Surely we still need to be able to display the correct string representation? @eanders

Reply:
In the HMIS, we do not show the number if we can't translate it. Since it's represented as a GraphQL Enum, we show `Invalid` if the value is not recognized as an enum value.  Which is maybe OK, I just don't know.